### ISSUE TITLE
TypeError in custom robots.txt editor

### ISSUE THREAD

**Issue by vena** [awesomemotive/all-in-one-seo-pack#3216]
With any combination of user-agent or directive, adding a value with more than one wildcard character, e.g. "/*?*test", throws an exception, disabling the robots.txt editor. it'll save, but the rule is no longer editable.

```
TypeError: this.errors.tableRule.find is not a function
    at Proxy.hasTableRuleError (.../wp-content/plugins/all-in-one-seo-pack/dist/Lite/assets/js/RobotsEditor.d5551e7e.js:4:4580)
    at .../wp-content/plugins/all-in-one-seo-pack/dist/Lite/assets/js/RobotsEditor.d5551e7e.js:4:14586
    at r (.../wp-content/plugins/all-in-one-seo-pack/dist/Lite/assets/js/vue.runtime.esm-bundler.588d3a9f.js:1:19524)
    at .../wp-content/plugins/all-in-one-seo-pack/dist/Lite/assets/js/Drag.bfb81d29.js:10:4476
    at Array.flatMap (<anonymous>)
    at Ce (.../wp-content/plugins/all-in-one-seo-pack/dist/Lite/assets/js/Drag.bfb81d29.js:10:4447)
    at Ft (.../wp-content/plugins/all-in-one-seo-pack/dist/Lite/assets/js/Drag.bfb81d29.js:10:4913)
    at Proxy.render (.../wp-content/plugins/all-in-one-seo-pack/dist/Lite/assets/js/Drag.bfb81d29.js:10:5994)
    at Gn (.../wp-content/plugins/all-in-one-seo-pack/dist/Lite/assets/js/vue.runtime.esm-bundler.588d3a9f.js:1:19850)
    at sn.I [as fn] (.../wp-content/plugins/all-in-one-seo-pack/dist/Lite/assets/js/vue.runtime.esm-bundler.588d3a9f.js:1:67094)
```

**Comment by arnaudbroes** [comment]
Hey @vena,

Thank you for reporting this issue. We have a very similar issue that we are already working on for our next release. I'll check with the relevant developer to see if his PR will also fix this issue.

Either way, we should be able to get this fixed in our 4.5.3 version which will be released next somewhere next month.

**Comment by arnaudbroes** [comment]
Hey @vena,

We've confirmed that that the PR I mentioned fixes this issue as well so as I mentioned earlier, this will be fixed in our 4.5.3 release which is scheduled for December 21st. If you're not able to wait until then, please let me know and I'll post a beta build here.
